src/modules/ui/routes.js

```diff
@@ -15,12 +16,15 @@ export default function (injectDeps, { clientStore, provider, domNode }) {
     return preview;
   };
 
+  const floatingMenu = () => (<FloatingMenu />);
+
   const root = (
     <div>
       <Layout
-        leftPanel={() => (<LeftPanel />)}
+        leftPanel={(menu) => (<LeftPanel floatingMenu={menu} />)}
```

Comment:
I don't like getting the menu from a function. Since we define the floatingMenu inside here. Let's use it directly.